lib/mix/tasks/dicon.deploy.ex

```diff
@@ -60,4 +62,16 @@ defmodule Mix.Tasks.Dicon.Deploy do
     Executor.exec(conn, command)
     Executor.exec(conn, ["rm ", release_file])
   end
+
+  defp write_custom_config(conn, name, target_dir, version) do
+    if config = host_config(name) do
+      content = :io_lib.format('~p.\n', [config])
+      target_sub_dir = ["/releases/", version, ?/]
+      config_sub_path = [target_sub_dir, "custom.config"]
+      Executor.write_file(conn, [target_dir, config_sub_path], content)
+      vm_args_path = [target_dir, target_sub_dir, "vm.args"]
+      content = ["-config ", ?., config_sub_path, ?\n]
+      Executor.write_file(conn, vm_args_path, content, :append)
+    end
```

Comment:
This is a subjective opinion ⚠️: I find this function slightly hard to follow because of the "wall-of-text" effect, so I'll suggest some layout edits that may make it more readable (they do IMO).

```elixir
defp write_custom_config(conn, name, target_dir, version) do
  if config = host_config(name) do
    # First, we compute the stuff we need for both writing custom.config and appending to vm.args
    target_sub_dir = ["/releases/", version, ?/]
    config_sub_path = [target_sub_dir, "custom.config"]

    # Then, after a good old newline, we write custom.config
    content = :io_lib.format('~p.\n', [config])
    Executor.write_file(conn, [target_dir, config_sub_path], content)

    # and in a "separate" paragraph, we write vm.args
    vm_args_path = [target_dir, target_sub_dir, "vm.args"]
    content = ["-config ", ?., config_sub_path, ?\n]
    Executor.write_file(conn, vm_args_path, content, :append)    
end
```

Wdyt?